model/src/test/kotlin/DependencyTreeNavigatorTest.kt

```diff
@@ -231,6 +232,17 @@ class DependencyTreeNavigatorTest : WordSpec() {
                 subProjectIds should containExactly(PROJECT_ID)
             }
         }
+
+        "dependencyTreeDepth" should {
+            "calculate the dependency tree depth for a project" {
+                navigator.dependencyTreeDepth(testProject, "compile") shouldBe 3
+                navigator.dependencyTreeDepth(testProject, "test") shouldBe 2
+            }
```

Comment:
I liked about the previous implementation of the test that it was self contained and also tested the `0` and `1` cases.

Reply:
Restored the original tests.